altair/examples/scatter_linked_table.py

```diff
@@ -0,0 +1,57 @@
+"""
+Brushing Scatter Plot to show data on a table
+---------------------------------
+A scatter plot of the cars dataset, with data tables for horsepower, MPG, and origin. 
+The tables update toreflect the selection on the scatter plot.
+"""
+# category: scatter plots
+
+import altair as alt
+from vega_datasets import data
+
+source = data.cars()
+
+# Brush for selection
+brush = alt.selection(type='interval')
+
+# Scatter Plot
+points = alt.Chart().mark_point().encode(
+    x='Horsepower:Q',
+    y='Miles_per_Gallon:Q',
+    color=alt.condition(brush, 'Cylinders:O', alt.value('grey'))
+).add_selection(brush)
+
+# Base chart for data tables
+ranked_text = alt.Chart().mark_text().encode(
+    y=alt.Y('row_number:O',axis=None)
+).transform_window(
+    row_number='row_number()'
+).transform_filter(
+    brush
+).transform_window(
+    rank='rank(row_number)'
+).transform_filter(
+    alt.datum.rank<20
+)
+
+# Data Tables
+horsepower = ranked_text.mark_text().encode(text='Horsepower:N')
+mpg = ranked_text.mark_text().encode(text='Miles_per_Gallon:N')
+origin = ranked_text.mark_text().encode(text='Origin:N')

```

Comment:
The ``mark_text()`` in each of these can be removed, because the ``ranked_text`` chart already specifies that.

Also, you can use the ``properties`` to set the title in this line; putting it all together it might look like

```python
text = alt.hconcat(
  ranked_text.encode(text='Horsepower:N').properties(title='Horsepower'),
  ranked_text.encode(text='Miles_per_Gallon:N').properties(title='MPG'),
  ranked_text.encode(text='Origin:N').properties(title='Origin'),
)
```

Reply:
Thanks! I thought there must be a better way to add the title, and I didn't realize I could layer encode in that way which will help with making my code less awkward all around. I'll try to make those changes (I'm an inexperienced contributor but will try to figure it out). Thanks again!